cmd/schemagen/schemagen.go

```diff
@@ -109,12 +109,16 @@ func renderTemplate(md *gocql.KeyspaceMetadata) ([]byte, error) {
 	}
 
 	imports := make([]string, 0)
+	if len(md.Types) != 0 {
+		imports = append(imports, "github.com/scylladb/gocqlx/v2")
+	}
+
 	for _, t := range md.Tables {
 		// Ensure ordered columns are sorted alphabetically
 		sort.Strings(t.OrderedColumns)
 
 		for _, c := range t.Columns {
-			if (c.Type == "timestamp" || c.Type == "date" || c.Type == "duration" || c.Type == "time") && !existsInSlice(imports, "time") {
```

Comment:
Why removing "duration" type here?

Reply:
i think it was leftover since time when duration was represented as time.Duration, as of now is getting converted into `gocql.Duration` and therefore no `time` package needed for it.